core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -4728,6 +4729,27 @@ public String toString() {
   //
   // FUSE integration related properties
   //
+  public static final PropertyKey FUSE_AUTH_POLICY_CLASS =
+      new Builder(Name.FUSE_AUTH_POLICY_CLASS)
+          .setDefaultValue("alluxio.fuse.auth.DefaultAuthPolicy")
```

Comment:
Can we not use `DefaultAuthPolicy` as the name, but use the content of the policy for the name so that people can know what each policy represents, e.g. `SystemUserGroupAuthPolicy`?

Reply:
Done